Hcal/testbeam04-2022/pedestals/testbeam22_run_info.csv

```diff
@@ -0,0 +1,58 @@
+# Made by hand May 16th, 2022
+# Time is in ISO format (YYYY-MM-DDTHH:MM)
```

Comment:
Move this to its own directory (maybe `Beam`) to separate it from the Hcal subsystem.